src/badge/Badge.story.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+import { text, withKnobs } from '@storybook/addon-knobs'
+import { Badge } from './index'
+
+<Meta title="Widgets|Badge" decorators={[withKnobs]}/>
```

Comment:
no need to declare knobs decorator here since it's already global

Reply:
Done